db/migrate/20181128140935_devise_token_auth_create_users.rb

```diff
@@ -0,0 +1,56 @@
+class DeviseTokenAuthCreateUsers < ActiveRecord::Migration[5.2]
+  def change
+    table = :users
+    change_table(table) do |t|
+      ## Required
+      t.string :provider, :null => false, :default => "email" unless column_exists?(table, :provider)
```

Comment:
Use the new syntax `null: false`

Reply:
Also there is no need to use the `unless` verification, we know the column exists

Reply:
Done!.